sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.15.0-beta.1 (Unreleased)
+## 1.14.1 (2024-10-10)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- [[#6064]](https://github.com/Azure/azure-sdk-for-cpp/issues/6064) Added internal support for the SDK packages to access more of telemetry features.
```

Comment:
We typically don't advertise internal only features in the changelog, since we don't want to encourage end customers to use things in that namespace. They are not guaranteed to be stable.

Reply:
It is for the customer to answer the question, "why am I upgrading the Core"? - "It has internal provisions for the packages for telemetry". Plus, should we release this change alone, it has to have some changelog entry, something has clearly changed, we should summarize what it is, and we are doing that, in vague terms, we are not encouraging to use `_internal`, we don't even mention it by name, as `_internal::UserAgentGeenrator`, for example. The contract hasn't changed - do not use `_internal` if you're a customer. I don't know how we can be more vague with these changelog entries, any ideas for better wording?

Reply:
> Plus, should we release this change alone, it has to have some changelog entry

We have other changes already so no need to make a CL entry for the sake of it (and we'd not ship a change like this alone).

Rather than being vague as a solution, I recommend removing this entry all together. We don't need to let customers know about internal only changes, it adds noise to the changelog that isn't relevant to them.

Reply:
Now that there's a "Bugs Fixed" entry, this change for Core is listed in "Other Changes", it will be less visible in web-published changelogs.